conda/plan.py

```diff
@@ -566,45 +560,11 @@ def get_resolve_object(index, prefix):
     return r
 
 
-def get_highest_priority_match(matches, prioritized_channel_list, index):
```

Comment:
This is not necessary anymore because `r.get_pkgs` now sorts by version and priority, and it throws a `PackageNotFound` error if there are no matches whatsoever.